Infusionsoft/InvoiceService.php

```diff
@@ -37,11 +37,16 @@ public static function chargeInvoiceArbitraryAmount($contactId, $invoiceId, $car
     public static function chargeInvoiceArbitraryAmountUsingPayPlan($invoiceId, $cardId, $amount, $merchantAccountId, $paymentNotes = 'API Arbitrary Payment', $numberOfPaymentsForChargePaymentPlan = 1, $daysBetweenPaymentsForChargePaymentPlan = 1){
 
         $result = false;
+        $today = new DateTime('now', new DateTimeZone('America/New_York'));
+        $today = $today->format('Y-m-d');

```

Comment:
Can you put a comment for why you are using the America/New_York timezone?  The commit says why, but it would be nice to have it here to, and explain that Infusionsoft is always on Eastern Time.